### PR TITLE
Ben: Added Parties to DB relation

### PR DESCRIPTION
**Pull request by benbianchi**
Added Parties to the models.

Each party has a name and a date. The Party also references a Fraternity. 

There can be many parties to one Fraternity.

### CODE REVIEW

File: ifc/party/models.py [hunk 1]
```diff
@@ -34,20 +35,20 @@ class Fraternity(SurrogatePK, Model):
 
     __tablename__ = 'fraternities'
```
Comment (Rdbaker) [inline ifc/party/models.py:36]: You have the same class defined twice here

Comment (Rdbaker) [inline ifc/party/models.py:36]: they also have the same (inaccurate) class docstring

File: migrations/versions/4fedf700f147_.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('fraternities', sa.Column('date', sa.Date(), nullable=False))
+    op.add_column('fraternities', sa.Column('fraternity_id', sa.Integer(), nullable=False))
+    op.create_foreign_key(None, 'fraternities', 'fraternities', ['fraternity_id'], ['id'])
```
Comment (Rdbaker) [inline migrations/versions/4fedf700f147_.py:21]: It looks like you're trying to add "parties", but you added "fraternities" instead.

File: ifc/templates/party/fraternityPartyList.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+{% extends "layout.html" %}
+
+{% block content %}
+<div class="body-content">
+    <div class="row">
+      <h1>Party's Registered for {{ user.fraternity }}</h1>
```
Comment (Rdbaker) [inline ifc/templates/party/fraternityPartyList.html:7]: there's a nice mixin that's automatically included called `current_user`, you should be using that instead

File: ifc/templates/party/fraternityPartyList.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+
```
Comment (Rdbaker) [inline ifc/templates/party/fraternityPartyList.html:1]: It looks like this file isn't being rendered anywhere, did you write this without seeing it?

File: ifc/templates/party/fraternityPartyList.html (hunk 3 above)
Comment (benbianchi) [inline ifc/templates/party/fraternityPartyList.html:7]: Done

File: ifc/templates/party/fraternityPartyList.html (hunk 4 above)
Comment (benbianchi) [inline ifc/templates/party/fraternityPartyList.html:1]: Yes, I dont know how or where to render it. I assume it's in the party,views.py file, right?

Comment (Rdbaker) [inline ifc/templates/party/fraternityPartyList.html:1]: Yes, that's the ideal place to call `render_template`. Take a look at the flask docs

**Pull request closed without merging**